styles/index-style.css

```diff
@@ -0,0 +1,58 @@
+h2 {
+    text-align: center;
+    font-size: 70px;
+}
+
+h3{
+    text-align: center;
+    font-size: 25px;
+}
+
+main {
+    grid-row: 2;
+    grid-column: 2/10;
+    display: grid;
+}
+main {
```

Comment:
Since these both target the same selector, the preference is to combine them into one rule set.